<ui version="4.0" >
 <class>DeclensionWidget</class>
 <widget class="QWidget" name="DeclensionWidget" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>723</width>
    <height>605</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize" >
   <size>
    <width>332</width>
    <height>326</height>
   </size>
  </property>
  <property name="windowTitle" >
   <string>-</string>
  </property>
  <layout class="QVBoxLayout" >
   <item>
    <widget class="QPushButton" name="makeNounButton" >
     <property name="text" >
      <string>Turn the selected word into a noun.</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="declensionGroupBox" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Preferred" hsizetype="Preferred" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title" >
      <string>Declension Forms</string>
     </property>
     <layout class="QGridLayout" >
      <item row="2" column="0" >
       <widget class="QLabel" name="singularFirstPersonLabel" >
        <property name="text" >
         <string comment="one of the seven cases (grammatical)" >Nominative</string>
        </property>
        <property name="buddy" >
         <cstring>singular_1</cstring>
        </property>
       </widget>
      </item>
      <item row="2" column="1" colspan="2" >
       <widget class="QLineEdit" name="singular_1" />
      </item>
      <item row="2" column="3" colspan="2" >
       <widget class="QLineEdit" name="dual_1" />
      </item>
      <item row="2" column="5" colspan="2" >
       <widget class="QLineEdit" name="plural_1" />
      </item>
      <item row="3" column="0" >
       <widget class="QLabel" name="singularSecondPersonLabel" >
        <property name="text" >
         <string comment="one of the seven cases (grammatical)" >Genitive</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1" colspan="2" >
       <widget class="QLineEdit" name="singular_2" />
      </item>
      <item row="3" column="3" colspan="2" >
       <widget class="QLineEdit" name="dual_2" />
      </item>
      <item row="3" column="5" colspan="2" >
       <widget class="QLineEdit" name="plural_2" />
      </item>
      <item row="4" column="0" >
       <widget class="QLabel" name="singularThirdMalePersonLabel" >
        <property name="text" >
         <string comment="one of the seven cases (grammatical)" >Dative</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1" colspan="2" >
       <widget class="QLineEdit" name="singular_3" />
      </item>
      <item row="4" column="3" colspan="2" >
       <widget class="QLineEdit" name="dual_3" />
      </item>
      <item row="4" column="5" colspan="2" >
       <widget class="QLineEdit" name="plural_3" />
      </item>
      <item row="5" column="0" >
       <widget class="QLabel" name="singularThirdFemalePersonLabel" >
        <property name="text" >
         <string comment="one of the seven cases (grammatical)" >Accusative</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1" colspan="2" >
       <widget class="QLineEdit" name="singular_4" />
      </item>
      <item row="5" column="3" colspan="2" >
       <widget class="QLineEdit" name="dual_4" />
      </item>
      <item row="5" column="5" colspan="2" >
       <widget class="QLineEdit" name="plural_4" />
      </item>
      <item row="6" column="0" >
       <widget class="QLabel" name="singularThirdNeutralPersonLabel" >
        <property name="text" >
         <string comment="one of the seven cases (grammatical)" >Ablative</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1" colspan="2" >
       <widget class="QLineEdit" name="singular_5" />
      </item>
      <item row="6" column="3" colspan="2" >
       <widget class="QLineEdit" name="dual_5" />
      </item>
      <item row="6" column="5" colspan="2" >
       <widget class="QLineEdit" name="plural_5" />
      </item>
      <item row="7" column="0" >
       <widget class="QLabel" name="label" >
        <property name="text" >
         <string comment="one of the seven cases (grammatical)" >Locative</string>
        </property>
       </widget>
      </item>
      <item row="7" column="1" colspan="2" >
       <widget class="QLineEdit" name="singular_6" />
      </item>
      <item row="7" column="3" colspan="2" >
       <widget class="QLineEdit" name="dual_6" />
      </item>
      <item row="7" column="5" colspan="2" >
       <widget class="QLineEdit" name="plural_6" />
      </item>
      <item row="8" column="0" >
       <widget class="QLabel" name="label_2" >
        <property name="text" >
         <string comment="one of the seven cases (grammatical)" >Vocative</string>
        </property>
       </widget>
      </item>
      <item row="8" column="1" colspan="2" >
       <widget class="QLineEdit" name="singular_7" />
      </item>
      <item row="8" column="3" colspan="2" >
       <widget class="QLineEdit" name="dual_7" />
      </item>
      <item row="8" column="5" colspan="2" >
       <widget class="QLineEdit" name="plural_7" />
      </item>
      <item row="1" column="1" colspan="2" >
       <widget class="QLabel" name="label_3" >
        <property name="text" >
         <string>Singular</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3" >
       <widget class="QLabel" name="label_4" >
        <property name="text" >
         <string>Dual</string>
        </property>
       </widget>
      </item>
      <item row="1" column="5" >
       <widget class="QLabel" name="label_5" >
        <property name="text" >
         <string>Plural</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <resources/>
 <connections/>
</ui>
